src/main/java/Cart.java

```diff
@@ -0,0 +1,74 @@
+package main.java;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Cart {
+    private final List<CartItem> cartItems; // избрани продукти
+    private double deliveryFee;
+
+    public Cart() {
+        this.cartItems = new ArrayList<>();
+    }
+
+    double getVAT() {
+        return this.calculateProductsPrices() * 0.2;
+    }
+
+
+    public double calculateFinalPrice() { //изчисляване на крайна цена на продуктите в кошницата заедно с ДДСто
```

Comment:
It's always better to describe your methods in JavaDoc instead of comments. You can of course use comments on places where you think It would be helpful, to clarify a statement for example.